syft/pkg/homebrew.go

```diff
@@ -0,0 +1,9 @@
+package pkg
+
+type HomebrewMetadata struct {
```

Comment:
I think the naming convention would make this `HomebrewEntry` -- `Entry` being a suffix that we use when there is a package manager with specific entries that we've read.